lib/pages/settings/model/request.dart

```diff
@@ -1,9 +1,23 @@
 class Request {
-  Request(this.userId, this.requestBody);
+  Request(this.userId, this.requestBody, this.processed, this.typeOfRequest);
 
   /// The user who created this request
   final String userId;
 
   /// The body of the request
   final String requestBody;
+
+  /// Boolean value represented whether the request has been processed by mods
+  final bool processed;
+
+  /// Type of the request
+  final RequestType typeOfRequest;
+}
+
+enum RequestType { permissions }
+
+extension ParseToString on RequestType {
+  String toShortString() {
+    return toString().split('.').last;
```

Comment:
Websites and events also have an (enum) type and I just call this directly, without a special method. Should we maybe add a generic extension on Object in the Utils class? What do you think?

Reply:
Why on Object and not only on Enum?